codingtest_ajw/3일차/문자열-다루기-기본-삼항연산자.js

```diff
@@ -0,0 +1,4 @@
+function solution(s) {
+    let answer = true;
+   return (s.length===4||s.length===6)&&isNaN(s)===false?true:false
```

Comment:
```suggestion
   return (s.length === 4 || s.length === 6) && !isNaN(s)
```
이렇게 바꿔도 무방합니당 물론 테스트케이스 11은 통과 못할거에요 ㅋㅋㅋ

Reply:
감사합니다!